mne/viz/tests/test_3d.py

```diff
@@ -230,32 +240,40 @@ def test_plot_alignment(tmpdir, renderer, mixed_fwd_cov_evoked):
     evoked_eeg_ecog_seeg.info['projs'] = []  # "remove" avg proj
     evoked_eeg_ecog_seeg.set_channel_types({'EEG 001': 'ecog',
                                             'EEG 002': 'seeg'})
-    with pytest.warns(RuntimeWarning, match='Cannot plot MEG'):
-        with catch_logging() as log:
-            plot_alignment(evoked_eeg_ecog_seeg.info, subject='sample',
-                           trans=trans_fname, subjects_dir=subjects_dir,
-                           surfaces=['white', 'outer_skin', 'outer_skull'],
-                           meg=['helmet', 'sensors'],
-                           eeg=['original', 'projected'], ecog=True, seeg=True,
-                           verbose=True)
+    with catch_logging() as log:
+        plot_alignment(evoked_eeg_ecog_seeg.info, subject='sample',
+                       trans=trans_fname, subjects_dir=subjects_dir,
+                       surfaces=['white', 'outer_skin', 'outer_skull'],
+                       meg=['helmet', 'sensors'],
+                       eeg=['original', 'projected'], ecog=True, seeg=True,
+                       verbose=True)
     log = log.getvalue()
-    assert '1 ECoG location' in log
-    assert '1 sEEG location' in log
+    assert 'ecog: 1' in log
+    assert 'seeg: 1' in log
     renderer.backend._close_all()
 
-    sphere = make_sphere_model(info=evoked.info, r0='auto', head_radius='auto')
+    sphere = make_sphere_model(info=info, r0='auto', head_radius='auto')
     bem_sol = read_bem_solution(op.join(subjects_dir, 'sample', 'bem',
                                         'sample-1280-1280-1280-bem-sol.fif'))
     bem_surfs = read_bem_surfaces(op.join(subjects_dir, 'sample', 'bem',
                                           'sample-1280-1280-1280-bem.fif'))
     sample_src[0]['coord_frame'] = 4  # hack for coverage
-    plot_alignment(info, subject='sample', eeg='projected',
-                   meg='helmet', bem=sphere, dig=True,
+    plot_alignment(info, trans_fname, subject='sample',
+                   eeg='projected', meg='helmet', bem=sphere, dig=True,
                    surfaces=['brain', 'inner_skull', 'outer_skull',
                              'outer_skin'])
```

Comment:
`trans` should not be required here, and shouldn't actually be used for that matter. A spherical BEM is defined in head coordinates actually... thus, as below, nothing here is in MRI coords.

Reply:
In this case the `trans` is required because the Freesurfer surfaces are in "mri" and we're plotting in "head"

Reply:
i.e. "brain", "inner_skull" etc.

Reply:
Those should not come from freesurfer but rather from the "bem" i.e. the sphere conductor model

Reply:
Ah okay, fixed. Yeah the way it was done previously, it was a bit unclear what was happening.